Pylint: W0612

```bash
mmpose/core/post_processing/group.py:251:8: W0612: Unused variable 'N' (unused-variable)
mmpose/core/post_processing/group.py:251:11: W0612: Unused variable 'K' (unused-variable)
mmpose/datasets/datasets/top_down/topdown_posetrack18_dataset.py:283:15: W0612: Unused variable 'preAll' (unused-variable)
mmpose/datasets/datasets/top_down/topdown_posetrack18_dataset.py:283:23: W0612: Unused variable 'recAll' (unused-variable)
mmpose/datasets/pipelines/bottom_up_transform.py:447:12: W0612: Unused variable 'idx' (unused-variable)
mmpose/datasets/pipelines/bottom_up_transform.py:449:27: W0612: Unused variable 'center' (unused-variable)
mmpose/datasets/pipelines/bottom_up_transform.py:449:35: W0612: Unused variable 'scale' (unused-variable)
mmpose/models/losses/mse_loss.py:68:12: W0612: Unused variable 'topk_val' (unused-variable)
mmpose/models/keypoint_heads/top_down_simple_head.py:139:12: W0612: Unused variable 'name' (unused-variable)
mmpose/models/keypoint_heads/bottom_up_simple_head.py:141:12: W0612: Unused variable 'name' (unused-variable)
mmpose/models/keypoint_heads/top_down_multi_stage_head.py:46:12: W0612: Unused variable 'i' (unused-variable)
mmpose/models/keypoint_heads/top_down_multi_stage_head.py:156:12: W0612: Unused variable 'name' (unused-variable)
mmpose/models/keypoint_heads/bottom_up_higher_resolution_head.py:187:12: W0612: Unused variable 'name' (unused-variable)
mmpose/models/detectors/bottom_up.py:304:12: W0612: Unused variable 'person_id' (unused-variable)
mmpose/models/detectors/top_down.py:320:16: W0612: Unused variable 'person_id' (unused-variable)
mmpose/models/backbones/hrnet.py:98:12: W0612: Unused variable 'i' (unused-variable)
mmpose/models/backbones/hrnet.py:440:12: W0612: Unused variable 'i' (unused-variable)
mmpose/models/backbones/resnet.py:379:16: W0612: Unused variable 'i' (unused-variable)
mmpose/models/mesh_heads/discriminator.py:283:8: W0612: Unused variable 'cams' (unused-variable)
mmpose/models/mesh_heads/hmr_head.py:76:12: W0612: Unused variable 'i' (unused-variable)
```

--- a/mmpose/core/post_processing/group.py
+++ b/mmpose/core/post_processing/group.py
@@ -248,7 +248,7 @@ class HeatmapParser:
             ans (list(np.ndarray)): Keypoint predictions.
             heatmaps (torch.Tensor[NxKxHxW]): Heatmaps.
         """
-        N, K, H, W = heatmaps.shape
+        _, _, H, W = heatmaps.shape
         for batch_id, people in enumerate(ans):
             for people_id, people_i in enumerate(people):
                 for joint_id, joint in enumerate(people_i):

--- a/mmpose/datasets/datasets/top_down/topdown_posetrack18_dataset.py
+++ b/mmpose/datasets/datasets/top_down/topdown_posetrack18_dataset.py
@@ -280,8 +280,7 @@ class TopDownPoseTrack18Dataset(TopDownCocoDataset):
         # evaluate per-frame multi-person pose estimation (AP)
         # compute AP
         print('Evaluation of per-frame multi-person pose estimation')
-        apAll, preAll, recAll = evaluateAP(gtFramesAll, prFramesAll, None,
-                                           False, False)
+        apAll, _, _ = evaluateAP(gtFramesAll, prFramesAll, None, False, False)
 
         # print AP
         print('Average Precision (AP) metric:')

--- a/mmpose/datasets/pipelines/bottom_up_transform.py
+++ b/mmpose/datasets/pipelines/bottom_up_transform.py
@@ -444,9 +444,9 @@ class BottomUpResizeAlign:
         test_scale_factor = results['ann_info']['test_scale_factor']
         aug_data = []
 
-        for idx, s in enumerate(sorted(test_scale_factor, reverse=True)):
+        for _, s in enumerate(sorted(test_scale_factor, reverse=True)):
             _results = results.copy()
-            image_resized, center, scale = _resize_align_multi_scale(
+            image_resized, _, _ = _resize_align_multi_scale(
                 _results['img'], input_size, s, min(test_scale_factor))
             _results['img'] = image_resized
             _results = self.transforms(_results)

--- a/mmpose/models/backbones/hrnet.py
+++ b/mmpose/models/backbones/hrnet.py
@@ -95,7 +95,7 @@ class HRModule(nn.Module):
                 conv_cfg=self.conv_cfg))
         self.in_channels[branch_index] = \
             num_channels[branch_index] * get_expansion(block)
-        for i in range(1, num_blocks[branch_index]):
+        for _ in range(1, num_blocks[branch_index]):
             layers.append(
                 block(
                     self.in_channels[branch_index],
@@ -437,7 +437,7 @@ class HRNet(nn.Module):
                 with_cp=self.with_cp,
                 norm_cfg=self.norm_cfg,
                 conv_cfg=self.conv_cfg))
-        for i in range(1, blocks):
+        for _ in range(1, blocks):
             layers.append(
                 block(
                     out_channels,

--- a/mmpose/models/backbones/resnet.py
+++ b/mmpose/models/backbones/resnet.py
@@ -376,7 +376,7 @@ class ResLayer(nn.Sequential):
                     norm_cfg=norm_cfg,
                     **kwargs))
             in_channels = out_channels
-            for i in range(1, num_blocks):
+            for _ in range(1, num_blocks):
                 layers.append(
                     block(
                         in_channels=in_channels,

--- a/mmpose/models/detectors/bottom_up.py
+++ b/mmpose/models/detectors/bottom_up.py
@@ -301,7 +301,7 @@ class BottomUp(BasePose):
         for res in result:
             pose_result.append(res['keypoints'])
 
-        for person_id, kpts in enumerate(pose_result):
+        for _, kpts in enumerate(pose_result):
             # draw each point on image
             if pose_kpt_color is not None:
                 assert len(pose_kpt_color) == len(kpts)

--- a/mmpose/models/detectors/top_down.py
+++ b/mmpose/models/detectors/top_down.py
@@ -317,7 +317,7 @@ class TopDown(BasePose):
                 wait_time=wait_time,
                 out_file=None)
 
-            for person_id, kpts in enumerate(pose_result):
+            for _, kpts in enumerate(pose_result):
                 # draw each point on image
                 if pose_kpt_color is not None:
                     assert len(pose_kpt_color) == len(kpts)

--- a/mmpose/models/keypoint_heads/bottom_up_higher_resolution_head.py
+++ b/mmpose/models/keypoint_heads/bottom_up_higher_resolution_head.py
@@ -184,11 +184,11 @@ class BottomUpHigherResolutionHead(nn.Module):
 
     def init_weights(self):
         """Initialize model weights."""
-        for name, m in self.deconv_layers.named_modules():
+        for _, m in self.deconv_layers.named_modules():
             if isinstance(m, nn.ConvTranspose2d):
                 normal_init(m, std=0.001)
             elif isinstance(m, nn.BatchNorm2d):
                 constant_init(m, 1)
-        for name, m in self.final_layers.named_modules():
+        for _, m in self.final_layers.named_modules():
             if isinstance(m, nn.Conv2d):
                 normal_init(m, std=0.001, bias=0)

--- a/mmpose/models/keypoint_heads/bottom_up_simple_head.py
+++ b/mmpose/models/keypoint_heads/bottom_up_simple_head.py
@@ -138,7 +138,7 @@ class BottomUpSimpleHead(nn.Module):
 
     def init_weights(self):
         """Initialize model weights."""
-        for name, m in self.deconv_layers.named_modules():
+        for _, m in self.deconv_layers.named_modules():
             if isinstance(m, nn.ConvTranspose2d):
                 normal_init(m, std=0.001)
             elif isinstance(m, nn.BatchNorm2d):

--- a/mmpose/models/keypoint_heads/top_down_multi_stage_head.py
+++ b/mmpose/models/keypoint_heads/top_down_multi_stage_head.py
@@ -43,7 +43,7 @@ class TopDownMultiStageHead(nn.Module):
 
         # build multi-stage deconv layers
         self.multi_deconv_layers = nn.ModuleList([])
-        for i in range(self.num_stages):
+        for _ in range(self.num_stages):
             if num_deconv_layers > 0:
                 deconv_layers = self._make_deconv_layer(
                     num_deconv_layers,
@@ -153,7 +153,7 @@ class TopDownMultiStageHead(nn.Module):
 
     def init_weights(self):
         """Initialize model weights."""
-        for name, m in self.multi_deconv_layers.named_modules():
+        for _, m in self.multi_deconv_layers.named_modules():
             if isinstance(m, nn.ConvTranspose2d):
                 normal_init(m, std=0.001)
             elif isinstance(m, nn.BatchNorm2d):

--- a/mmpose/models/keypoint_heads/top_down_simple_head.py
+++ b/mmpose/models/keypoint_heads/top_down_simple_head.py
@@ -136,7 +136,7 @@ class TopDownSimpleHead(nn.Module):
 
     def init_weights(self):
         """Initialize model weights."""
-        for name, m in self.deconv_layers.named_modules():
+        for _, m in self.deconv_layers.named_modules():
             if isinstance(m, nn.ConvTranspose2d):
                 normal_init(m, std=0.001)
             elif isinstance(m, nn.BatchNorm2d):

--- a/mmpose/models/losses/mse_loss.py
+++ b/mmpose/models/losses/mse_loss.py
@@ -65,7 +65,7 @@ class JointsOHKMMSELoss(nn.Module):
         N = len(loss)
         for i in range(N):
             sub_loss = loss[i]
-            topk_val, topk_idx = torch.topk(
+            _, topk_idx = torch.topk(
                 sub_loss, k=self.topk, dim=0, sorted=False)
             tmp_loss = torch.gather(sub_loss, 0, topk_idx)
             ohkm_loss += torch.sum(tmp_loss) / self.topk

--- a/mmpose/models/mesh_heads/discriminator.py
+++ b/mmpose/models/mesh_heads/discriminator.py
@@ -280,7 +280,7 @@ class SMPLDiscriminator(nn.Module):
 
     def forward(self, thetas):
         """Forward function."""
-        cams, poses, shapes = thetas
+        _, poses, shapes = thetas
 
         batch_size = poses.shape[0]
         shape_disc_value = self.shape_discriminator(shapes)

--- a/mmpose/models/mesh_heads/hmr_head.py
+++ b/mmpose/models/mesh_heads/hmr_head.py
@@ -73,7 +73,7 @@ class MeshHMRHead(nn.Module):
         pred_pose = init_pose
         pred_shape = init_shape
         pred_cam = init_cam
-        for i in range(self.n_iter):
+        for _ in range(self.n_iter):
             xc = torch.cat([x, pred_pose, pred_shape, pred_cam], 1)
             xc = self.fc1(xc)
             xc = self.drop1(xc)

